src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User user = UserService.findByEmail(email);
+        if (user != null) {
+            if (user.getPassword().equals(password)) {
+                return true;
+            }
+        }
+
         return false;
```

Comment:
we don't need if/else construction to return boolean value
```suggestion
        return user != null && user.getPassword().equals(password);
```